.github/workflows/php.yaml

```diff
@@ -0,0 +1,60 @@
+name: PHP CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  php:
+    name: PHP project (PHP ${{ matrix.php-versions }})
+
+    runs-on: ubuntu-latest
+    if: "! contains(toJSON(github.event.commits.*.message), '[skip ci]')"
```

Comment:
:eyes:

Reply:
I agree... This is suspicious... I'll remove it